components/automate-ui/src/app/page-components/profile/profile.component.scss

```diff
@@ -1,107 +1,96 @@
 @import "~styles/variables";
 @import "~styles/mixins";
 
-.profile,
+$dropdown-width: 196px;
+
 button {
   -webkit-appearance: none;
-  background: transparent;
```

Comment:
Button seems to be gray in Firefox w/o this line. Not sure atm why its handling the background differently.

<img width="277" alt="Screen Shot 2019-05-02 at 11 17 23 PM" src="https://user-images.githubusercontent.com/479121/57120776-c6a3eb80-6d30-11e9-8d23-7e839de2d8e9.png">